epoch/api/contract_api_usage.md

```diff
@@ -0,0 +1,45 @@
+[back](./README.md)
+# Contracts - intended usage
+
+## Introduction
+You interact with a Aeternity node through HTTP.
+To learn more about contracts and contract life cycles see [the doc](/contracts/contracts.md).
+
+The following assumes that the node exposes at address 127.0.0.1 the following ports:
+* HTTP external API: 3013
+* HTTP internal API: 3113
+
+There are two basic types of API calls, off-chain operations and on-chain transactions.
+
+In order to affect the state of the chain you have to submit a signed transaction
+to a mining node.
+
+For an up to date list of all HTTP API endpoints and their arguments see
+[Epoch HTTP API](https://aeternity.github.io/epoch-api-docs/?config=https://raw.githubusercontent.com/aeternity/epoch/master/apps/aehttp/priv/swagger.json)
```

Comment:
I suggest removing this paragraph as it links to master - so breaks tagging - and anyway list of APIs shall be linked from release notes.

Reply:
I think having links from protocol/epoch API descriptions to the generated specific API documentation is a good thing, but there should perhaps be a nice way to link to the documentation of the latest release.